packages/common/styles/src/lib/notional-theme/notional-theme.ts

```diff
@@ -30,7 +30,7 @@ export const colors = {
   purpleGrey: '#E7E8F2',
   lightGrey: '#E6EAEB',
   secondaryGrey: '#8F9BB3',
-  greenGrey: '#95B2BA',
+  greenGrey: '#BCD4DB',
```

Comment:
Is this right? Should we be changing a color in here like this?

Reply:
Yes, this came from Isaac. Apparently he thought we had updated this greenGrey color a while ago